ibis/backends/pandas/executor/generic.py

```diff
@@ -0,0 +1,319 @@
+from __future__ import annotations
+
+import decimal
+import operator
+
+import numpy as np
+import pandas as pd
+
+import ibis.expr.operations as ops
+from ibis.backends.pandas.convert import PandasConverter
+from ibis.backends.pandas.executor.core import execute
+from ibis.backends.pandas.executor.utils import asframe, columnwise
+from ibis.common.exceptions import OperationNotDefinedError
+
+
+@execute.register(ops.Literal)
+def execute_literal(op, value, dtype):
+    if dtype.is_interval():
+        value = pd.Timedelta(value, dtype.unit.short)
+    elif dtype.is_array():
+        value = np.array(value)
```

Comment:
Shouldn't this be `list(value)`?

Reply:
It is working for now, we can revisit once the pandas out put format has clear guidelines about the result types.